main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -194,6 +194,7 @@ private static void migrateSettings() {
             e.putInt(getKey(R.string.pref_defaultNavigationTool), prefsV0.getInt(getKey(R.string.pref_defaultNavigationTool), NavigationAppsEnum.COMPASS.id));
             e.putInt(getKey(R.string.pref_defaultNavigationTool2), prefsV0.getInt(getKey(R.string.pref_defaultNavigationTool2), NavigationAppsEnum.INTERNAL_MAP.id));
             e.putBoolean(getKey(R.string.pref_debug), prefsV0.getBoolean(getKey(R.string.pref_debug), false));
+            e.putBoolean(getKey(R.string.pref_global_wp_extraction_disable), prefsV0.getBoolean(getKey(R.string.pref_global_wp_extraction_disable), false));
```

Comment:
IMHO this line is not needed, as this setting is new, so no need to migrate.

Reply:
I wasn't sure ;) Now I have removed it.